### PR TITLE
Clean up field definitions

### LINKED ISSUE #4450: Fill descriptions for fields without
Fields with missing descriptions can be found here: https://github.com/jalvz/apm-server/tree/generate-package-fields/apmpackage/apm/0.1.0

### PR DESCRIPTION
**Pull request by axw**
## Motivation/summary

Update all `fields.yml` files, adding a description to all leaf fields. Escape special characters in the generated `README.md`, to fix some rendering issues.

While adding descriptions I noticed some fields were misplaced. e.g. breakdown metric fields in the "error", "transaction", and "span" fields. I also removed the old "view spans" field which I believe was only used a long time ago in dashboards but is no longer required.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~ Does not affect released, user-facing, functionality.
- [x] Documentation has been updated

## How to test these changes

Check all fields have a description in the integration package README.

## Related issues

Closes #4450

### CODE REVIEW

File: apmpackage/apm/0.1.0/data_stream/internal_metrics/fields/fields.yml [hunk 1]
```diff
@@ -153,8 +163,11 @@
     Transactions that are 'sampled' will include all available information. Transactions that are not sampled will not have spans or context.
 - name: transaction.self_time.count
   type: long
+  description: Number of aggregated transactions.
 - name: transaction.self_time.sum.us
   type: long
+  description: |
+    Aggregated transaction durations, excluding the durations where a direct child was running, in microseconds.
```
Comment (simitt) [inline apmpackage/apm/0.1.0/data_stream/internal_metrics/fields/fields.yml:170]: nit: in https://github.com/elastic/apm-server/pull/4689/files#diff-8d8bd845e531a70d1d9c1559419ef257df67122295376080d90f3610442764d1R148 you use singular, here plural for `durations`. Would you mind aligning?

**Review by simitt — APPROVED** [review]
Awesome!

File: apmpackage/apm/0.1.0/data_stream/internal_metrics/fields/fields.yml (hunk 1 above)
Comment (axw) [reply apmpackage/apm/0.1.0/data_stream/internal_metrics/fields/fields.yml:170]: Done, made them singular. I did change the second "durations" to "time periods", to clarify that the time periods are being excluded from the aggregated duration.

**Comment by axw** [comment]
Test failure is unrelated, see https://github.com/elastic/apm-server/issues/4692

**Comment by simitt** [comment]
Checked the [exported fields docs](https://www.elastic.co/guide/en/apm/server/master/exported-fields.html) and the _Exported Fields_ section when installing the APM integration.

**Pull request merged**